modules/nw-sriov-network-operator.adoc

```diff
@@ -0,0 +1,94 @@
+The Single Root I/O Virtualization (SR-IOV) network operator is used to allocate and configure Virtual Functions (VFs) from SR-IOV-supporting Physical Function NICs on the nodes.
+
+For more information on deploying the operator, see _Installing the SR-IOV Network Operator_.
+For more information on configuring an SR-IOV network device, see _Configuring an SR-IOV network device_.
+.Procedure
+
+There are some differences between running Data Plane Development Kit (DPDK) workloads on Intel VFs or Mellanox VFs. This section provides object configuration examples for both VF types.
+The following is an example of an `sriovNetworkNodePolicy` object used to run DPDK applications on Intel NICs:
+[source,yaml]
+----
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetworkNodePolicy
+    metadata:
+     name: dpdk-nic-1
+     namespace: openshift-sriov-network-operator
+    spec:
+     deviceType: vfio-pci <1>
+     needVhostNet: true <2>
+     nicSelector:
+       pfNames: ["ens3f0"]
+     nodeSelector:
+       node-role.kubernetes.io/worker-cnf: ""
+     numVfs: 10
+     priority: 99
+     resourceName: dpdk_nic_1
+    ---
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetworkNodePolicy
+    metadata:
+     name: dpdk-nic-1
+     namespace: openshift-sriov-network-operator
+    spec:
+     deviceType: vfio-pci
+     needVhostNet: true
+     nicSelector:
+         pfNames: ["ens3f1"]
+     nodeSelector:
+     node-role.kubernetes.io/worker-cnf: ""
+     numVfs: 10
+     priority: 99
+     resourceName: dpdk_nic_2
+
+----
+<1> For Intel NICs, `deviceType` must be `vfio-pci`.
+<2> If kernel communication with DPDK workloads is required, add `needVhostNet: true`. This mounts the `/dev/net/tun` and `/dev/vhost-net` devices into the container so the application can create a tap device and connect the tap device to the DPDK workload.
+
+The following is an example of an `sriovNetworkNodePolicy` object for Mellanox NICs:
+[source,yaml]
+----
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetworkNodePolicy
+    metadata:
+     name: dpdk-nic-1
+     namespace: openshift-sriov-network-operator
+    spec:
+     deviceType: netdevice <1>
+     isRdma: true <2>
+     nicSelector:
+       rootDevices:
+         - "0000:5e:00.1"
+     nodeSelector:
+       node-role.kubernetes.io/worker-cnf: ""
+     numVfs: 5
+     priority: 99
+     resourceName: dpdk_nic_1
+    ---
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetworkNodePolicy
+    metadata:
+     name: dpdk-nic-2
+     namespace: openshift-sriov-network-operator
+    spec:
+     deviceType: netdevice
+     isRdma: true
+     nicSelector:
+       rootDevices:
+         - "0000:5e:00.0"
+     nodeSelector:
+       node-role.kubernetes.io/worker-cnf: ""
+     numVfs: 5
+     priority: 99
+     resourceName: dpdk_nic_2
+----
+<1> For Mellanox devices the `deviceType` must be `netdevice`.
+<2> For Mellanox devices `isRDMA` must be `true`.
```

Comment:
In the code block, the parameter is stated as `isRdma`. You could update the reference in this call out line to use the same case.

Reply:
Done.